packages/react-devtools-shared/src/__tests__/inspectedElementContext-test.js

```diff
@@ -158,7 +158,7 @@ describe('InspectedElementContext', () => {
     }
 
     const ModernContext = React.createContext();
-    ModernContext.displayName = 'ModernContext';
+    (ModernContext: any).displayName = 'ModernContext';
```

Comment:
I guess I should just add `displayName` to our internal context type. Will do momentarily.